cmd/daemon/flux-notification-controller/server.go

```diff
@@ -0,0 +1,15 @@
+package flux_notification_controller
+
+import (
+	"net/http"
+
+	"github.com/gorilla/mux"
+	"github.com/lunarway/release-manager/internal/log"
+)
+
+func StartHttpServer() {
+	router := mux.NewRouter()
+	router.HandleFunc("/webhook/flux2-alerts", HandleEventFromFlux2).Methods(http.MethodPost)
+	err := http.ListenAndServe(":3001", router)
+	log.Errorf("Failed to start daemon's HTTP server: %v", err)
```

Comment:
To avoid logging `Failed to start daemon's HTTP server: nil` when the server stops we need a condition around this.

```suggestion
	if err != nil {
		log.Errorf("Failed to start daemon's HTTP server: %v", err)
	}
```

Reply:
🙈